src/ui/components/shared/UserSettingsModal/ExperimentalSettings.tsx

```diff
@@ -56,6 +56,11 @@ const RISKY_EXPERIMENTAL_SETTINGS: ExperimentalSetting[] = [
     description: "Supports replaying longer recordings",
     key: "tenMinuteReplays",
   },
+  {
+    label: "Advanced timeline",
+    description: "Show loading, loaded, and indexed regions",
+    key: "advancedTimeline",
+  },
```

Comment:
We should not expose this feature to users. this is one of those `window.app.prefs` features.